### PR TITLE
SW-5981 Project not showing up under Participant when creating a new document

### PR DESCRIPTION
**Pull request by constanzauanini**

### CODE REVIEW

**Review by tommylau523 — APPROVED** [review]

File: src/components/DocumentProducer/DocumentMetadataEdit/index.tsx [hunk 1]
```diff
@@ -161,7 +163,10 @@ const DocumentMetadataEdit = ({
 
       <FormField>
         <ProjectsDropdown
-          availableProjects={participantProjects}
+          availableProjects={participantProjects?.map((partProj) => ({
```
Comment (tommylau523) [inline src/components/DocumentProducer/DocumentMetadataEdit/index.tsx:166]: nit: `partProj` can just be `project`

File: src/components/ProjectsDropdown/index.tsx [hunk 2]
```diff
@@ -4,12 +4,17 @@ import { Dropdown, DropdownItem } from '@terraware/web-components';
 
 import { useLocalization } from 'src/providers';
 import strings from 'src/strings';
-import { Project } from 'src/types/Project';
+
+type AvailableProject = {
+  name: string;
+  id: number;
+  [key: string]: any;
```
Comment (tommylau523) [inline src/components/ProjectsDropdown/index.tsx:11]: Do we need this here? Seems like ProjectDropdown will not actually use them

File: src/components/ProjectsDropdown/index.tsx [hunk 3]
```diff
@@ -4,12 +4,17 @@ import { Dropdown, DropdownItem } from '@terraware/web-components';
 
 import { useLocalization } from 'src/providers';
 import strings from 'src/strings';
-import { Project } from 'src/types/Project';
+
+type AvailableProject = {
```
Comment (tommylau523) [inline src/components/ProjectsDropdown/index.tsx:8]: Can rename this to ProjectDropdownOption to be consistent with our DropdownOption namings!

**Review by tommylau523 — CHANGES_REQUESTED** [review]
Some minor naming stuff!

File: src/components/ProjectsDropdown/index.tsx (hunk 2 above)
Comment (constanzauanini) [reply src/components/ProjectsDropdown/index.tsx]: No, I think I was just struggling with typescript! Thanks

**Review by tommylau523 — APPROVED** [review]

**Pull request merged**